test/apps/iodemo/io_demo.cc

```diff
@@ -2736,6 +2750,7 @@ static int parse_args(int argc, char **argv, options_t *test_opts)
 #endif
                       << std::endl;
             std::cout << "  -L <progress_count>         Maximal number of consecutive ucp_worker_progress invocations" << std::endl;
+            std::cout << "  -I <src_addr>               Set source address to select device" << std::endl;
```

Comment:
maybe: "Set source IP address to select network interface on client side"?

Reply:
fixed and added empty line after each }